flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/table/DynamoDbConnectorOptions.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.connector.dynamodb.table;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+/**
+ * Base options for the DynamoDb connector. Needs to be public so that the {@link
+ * org.apache.flink.table.api.TableDescriptor} can access it.
+ */
+public class DynamoDbConnectorOptions {
+
+    private DynamoDbConnectorOptions() {
+        // private constructor to prevent initialization of static class
+    }
+
+    public static final ConfigOption<String> TABLE_NAME =
+            ConfigOptions.key("table")
```

Comment:
Should we keep this consistent with the DataStream field? `destination-table-name` or rename both to `table-name`?